### PR TITLE
chore(log):add more visible log and status for evaluation

### LINKED ISSUE #1775: pod ImagePullBackOff but task status is still running
**Describe the bug**
image
image
task status is still running and it does not create new pods or kill itself.

**To Reproduce**
follow https://doc.starwhale.ai/zh/docs/quickstart/on-premises doc.

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Description

1. add more visible flags for log on the client
2. add more statuses for task on the server(closes #1775 )

## Modules
- [ ] UI
- [x] Controller
- [x] Client
- [ ] Python-SDK
- [x] Others

## Checklist
- [ ] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

**Comment by tianweidut** [comment]
LGTM

File: server/controller/src/main/java/ai/starwhale/mlops/schedule/k8s/PodEventHandler.java [hunk 1]
```diff
@@ -47,33 +52,111 @@ public void onAdd(V1Pod obj) {
...
-        if (CollectionUtils.isEmpty(optionalTasks)) {
-            log.warn("no tasks found for pod {}", newObj.getMetadata().getName());
+        Long tid = getTaskId(pod);
+        if (tid != null) {
+            TaskStatus taskStatus = statusOf(pod.getStatus().getPhase());
+            if (taskStatus == TaskStatus.UNKNOWN) {
+                return;
+            }
+            log.debug("task:{} status changed to {}.", tid, taskStatus);
+            taskStatusReceiver.receive(List.of(new ReportedTask(tid, taskStatus)));
```
Comment (anda-ren) [inline server/controller/src/main/java/ai/starwhale/mlops/schedule/k8s/PodEventHandler.java:137]: watcher on k8s pod would make sw job failed once after the very first pod failing. The watcher on k8s job would make sw job failed after the k8s job failing after several pod restarts.

**Review by anda-ren — CHANGES_REQUESTED** [review]

File: server/controller/src/main/java/ai/starwhale/mlops/schedule/k8s/PodEventHandler.java (hunk 1 above)
Comment (goldenxinxing) [reply server/controller/src/main/java/ai/starwhale/mlops/schedule/k8s/PodEventHandler.java]: fixed

**Review by anda-ren — APPROVED** [review]
LGTM

**Pull request merged**